secv_guis/bimask_app/main_window.py

```diff
@@ -71,6 +71,8 @@ def exp_threshold(keep_p_value, lmbd):
 
 
 def pmap_to_mask(pmap, keep_highest_pval=0.05, discard_lowest_pval=0.5):
+    print(np.max(pmap),np.min(pmap),keep_highest_pval)
+    return (pmap/(2**16-1))>=keep_highest_pval
```

Comment:
what is this? I don't understand why would you divide a probability map by 2**16. If this what you refer to as "Fixed normalization for preannotations", I don't see how that happens.

In general please update the docstrings with the new behaviour/expectations, otherwise the code breaks and becomes very difficult to use/interpret/maintain.

Also, is this code really returning before the docstring? That should not happen. The expected order is `def(parameters): """docstring""" body return result`

Reply:
I forget to get rid of my testing commit.
fixed later

Reply:
Sorry for the delay in my answer, busy days.

And thanks a lot for your detailed explanations, it all sounds fine. I'm just missing this 1 bit of dividing by 2**16. Could you explain what was the idea here? I'll be happy to accept your PR and put a couple of commits on top to clean it up.

I'll have to do that anyway since, as you can see in the [Travis CI](https://travis-ci.org/github/andres-fr/secv-guis/pull_requests) job, some things in the PR are failing.

Reply:
The CI is failing on the unit tests due to some dependency issues (I didn't specify the versions, and current ones seems to be clashing). That has nothing to do with your contribution, I'll have to sort it out.

The CI is also failing on the code style. Just FYI, if you check the logs it tells you what errors it found (you can also see those if you run `flake8` on your project yourself. Also, not a big deal, I'll have to check style myself anyway so just mentioning in case this info is useful to you. Linting is not only about style, it also catches programming errors (e.g. when you have defined unused variables)

```
$ python -m flake8

./secv_guis/base_widgets.py:51:34: E231 missing whitespace after ','

./secv_guis/base_widgets.py:54:80: E501 line too long (88 > 79 characters)

./secv_guis/base_widgets.py:318:27: E225 missing whitespace around operator

./secv_guis/base_widgets.py:335:61: E231 missing whitespace after ','

./secv_guis/base_widgets.py:337:61: E231 missing whitespace after ','

./secv_guis/base_widgets.py:339:51: E231 missing whitespace after ','

./secv_guis/base_widgets.py:339:55: E231 missing whitespace after ','

./secv_guis/base_widgets.py:341:52: E231 missing whitespace after ','

./secv_guis/base_widgets.py:341:56: E231 missing whitespace after ','

./secv_guis/base_widgets.py:343:47: E231 missing whitespace after ','

./secv_guis/base_widgets.py:344:49: E231 missing whitespace after ','

./secv_guis/base_widgets.py:382:44: E231 missing whitespace after ','

./secv_guis/base_widgets.py:394:50: E231 missing whitespace after ','

./secv_guis/base_widgets.py:397:49: E231 missing whitespace after ','

./secv_guis/bimask_app/main_window.py:20:1: F401 'skimage.filters.apply_hysteresis_threshold' imported but unused

./secv_guis/bimask_app/main_window.py:35:22: E231 missing whitespace after ','

./secv_guis/bimask_app/main_window.py:35:39: E231 missing whitespace after ','

./secv_guis/bimask_app/main_window.py:35:56: E231 missing whitespace after ','

./secv_guis/bimask_app/main_window.py:36:9: E225 missing whitespace around operator

./secv_guis/bimask_app/main_window.py:38:50: E225 missing whitespace around operator

./secv_guis/bimask_app/main_window.py:39:7: E225 missing whitespace around operator

./secv_guis/bimask_app/main_window.py:39:64: E202 whitespace before ')'

./secv_guis/bimask_app/main_window.py:40:14: E225 missing whitespace around operator

./secv_guis/bimask_app/main_window.py:40:26: E225 missing whitespace around operator

./secv_guis/bimask_app/main_window.py:41:14: E225 missing whitespace around operator

./secv_guis/bimask_app/main_window.py:41:26: E225 missing whitespace around operator

./secv_guis/bimask_app/main_window.py:42:9: E225 missing whitespace around operator

./secv_guis/bimask_app/main_window.py:42:14: E225 missing whitespace around operator

./secv_guis/bimask_app/main_window.py:48:1: E302 expected 2 blank lines, found 1

./secv_guis/bimask_app/main_window.py:215:74: W291 trailing whitespace

./secv_guis/bimask_app/main_window.py:223:16: E225 missing whitespace around operator

./secv_guis/bimask_app/main_window.py:224:30: E225 missing whitespace around operator

./secv_guis/bimask_app/main_window.py:225:20: E225 missing whitespace around operator

./secv_guis/bimask_app/main_window.py:225:26: E231 missing whitespace after ','

./secv_guis/bimask_app/main_window.py:225:28: E231 missing whitespace after ','

./secv_guis/bimask_app/main_window.py:227:22: E225 missing whitespace around operator

./secv_guis/bimask_app/main_window.py:231:80: E501 line too long (85 > 79 characters)

./secv_guis/bimask_app/main_window.py:269:54: W291 trailing whitespace

./secv_guis/bimask_app/main_window.py:270:33: E128 continuation line under-indented for visual indent

./secv_guis/bimask_app/main_window.py:271:33: E128 continuation line under-indented for visual indent

./secv_guis/bimask_app/main_window.py:469:62: E231 missing whitespace after ','

./secv_guis/bimask_app/main_window.py:707:80: E501 line too long (92 > 79 characters)

./secv_guis/bimask_app/main_window.py:708:60: E127 continuation line over-indented for visual indent

./secv_guis/bimask_app/main_window.py:710:80: E501 line too long (84 > 79 characters)

./secv_guis/bimask_app/main_window.py:711:60: E127 continuation line over-indented for visual indent

./secv_guis/bimask_app/main_window.py:730:9: F841 local variable 'pval' is assigned to but never used
```